src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,25 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: $main-padding;

```

Comment:
Don't be greedy, add some top paddings, so the top row don't go over top of the page on hover